server/sonar-web/src/main/js/apps/projects/components/PageSidebar.js

```diff
@@ -75,8 +86,77 @@ export default function PageSidebar({
         <SearchFilterContainer query={query} isFavorite={isFavorite} organization={organization} />
       </div>
       <QualityGateFilter query={query} isFavorite={isFavorite} organization={organization} />
-      {!isLeakView &&
-        <PageSidebarOverall query={query} isFavorite={isFavorite} organization={organization} />}
+      {!isLeakView && [
+        <ReliabilityFilter
```

Comment:
Because there are many similar facets, I suggest to do:
```javascript
const facetProps = { query, isFavorite, organization };
...
<ReliabilityFilter key="reliability" {...facetProps} />
<SecurityFilter key="security" {...facetProps} />
```

In this case each facet takes only one line of code, and the readability increases.